server_admin/topics/users/viewing.adoc

```diff
@@ -1,19 +1,24 @@
+[searching-users]
+= Searching for users
 
-=== Searching For Users
+You can search for a user in the database, by using the full name, last name, or email address of the user.
```

Comment:
remove " .., by". 
So you cannot search by first name?

Reply:
You can search by first name.